getCalspec/rebuild.py

```diff
@@ -43,9 +45,24 @@ def add_astroquery_id(df):
 
 
 def add_alt_star_name(df):
```

Comment:
Some sort of minimal docstrings, even if you don't want to do them thoroughly, would be really useful, even just to help guide the reviewer/our future selves. Something like `Operates on the dataframe in-place, adding the alternate names for each star (row), and removes spaces from HD stars.`